rust/osm2lanes/src/transform/tags_to_lanes/modes/bus.rs

```diff
@@ -255,6 +296,7 @@ fn lanes_bus(
     Ok(())
 }
 
+//noinspection ALL
```

Comment:
hmm

Reply:
Whoops, this was a side-effect of trying to fix that reported error. Well spotted.